lib/kafo/help_builders/base.rb

```diff
@@ -4,14 +4,17 @@ module Kafo
   module HelpBuilders
     DEFAULT_GROUP_NAME   = 'Basic'
     DEFAULT_MODULE_NAME  = 'Generic'
+    ADVANCED_MODULE_NAME = 'Advanced'
     IGNORE_IN_GROUP_NAME = /\s*parameters:?/
 
     class Base < Clamp::Help::Builder
       include StringHelper
 
-      def initialize(params)
+      def initialize(params, app_options, advanced: false)
```

Comment:
Perhaps it should keep the API compatible?
```suggestion
      def initialize(params, app_options = {}, advanced: false)
```